app/models/concerns/ccms_submission_state_machine.rb

```diff
@@ -34,12 +36,21 @@ module CCMSSubmissionStateMachine
         transitions from: :case_submitted, to: :case_created
       end
 
+      event :obtain_document_ids do
+        transitions from: :case_created, to: :document_ids_obtained
+      end
+
+      event :complete do
+        transitions from: :case_created, to: :completed
+      end
```

Comment:
Shouldn't this be `transitions from: :document_ids_obtained, to: :completed` ?

Reply:
You're right that there will be another transition from :document_ids_obtained to: :completed, but I'll do that in the next story. This one covers the situation where there are no documents, so the status moves straight from :case_created to :completed